spec/system/home_spec.rb

```diff
@@ -174,4 +174,26 @@ def ie_alert_box_xpath
 
     expect(page).not_to have_css(".title", text: "Featured")
   end
+
+  describe "Header Card" do
+    scenario "if there is header card with link, the link content is rendered" do
+      create(:widget_card,
+        title: "Card text",
+        description: "Card description",
+        link_text: "Link text",
+        link_url: "consul.dev",
+        header: true
+      )
```

Comment:
The best way to follow the line length rule is to write less code :stuck_out_tongue_winking_eye:.

```suggestion
      create(:widget_card, :header, link_text: "Link text", link_url: "consul.dev")
```